wire-compiler/src/test/java/com/squareup/wire/WireCompilerErrorTest.kt

```diff
@@ -119,6 +119,30 @@ class WireCompilerErrorTest {
           """.trimMargin())
   }
 
+  @Test
+  fun testEnumNamespaceFile() {
+    val e = assertFailsWith<SchemaException> {
+      compile("""
+          |package com.squareup.protos.test;
+          |
+          |enum Bar {
+          |  QUIX = 0;
```

Comment:
FWIW, `0` is a terrible value for an enum constant with protos

Reply:
Because of proto3 absence of absence, right? Or even in proto2?

Reply:
Yeah.